src/app/ui/colors.ts

```diff
@@ -0,0 +1,24 @@
+export const colors = {
```

Comment:
any colors added to this will be populated as css vars `--color-[key]` and change whenever the user changes their preference of changes the system color theme.